dashboards-observability/public/components/explorer/explorer.tsx

```diff
@@ -109,20 +118,35 @@ export const Explorer = ({
   const [timeIntervalOptions, setTimeIntervalOptions] = useState(TIME_INTERVAL_OPTIONS);
   const [isOverridingTimestamp, setIsOverridingTimestamp] = useState(false);
   const [tempQuery, setTempQuery] = useState(query[RAW_QUERY]);
+  const [isLiveTailPopoverOpen, setIsLiveTailPopoverOpen] = useState(false);
+  const [isLiveTailOn, setIsLiveTailOn] = useState(false);
+  const [liveTailTabId, setLiveTailTabId] = useState(TAB_EVENT_ID);
+  const [liveTailName, setLiveTailName] = useState('');
+  const [liveTailToggle, setLiveTailToggle] = useState(false);
+
   
   const queryRef = useRef();
   const selectedPanelNameRef = useRef('');
   const explorerFieldsRef = useRef();
+  const isLiveTailOnRef = useRef(true);
+  const liveTailTabIdRef = useRef('');
+  const liveTailNameRef = useRef('');
   queryRef.current = query;
   selectedPanelNameRef.current = selectedPanelName;
   explorerFieldsRef.current = explorerFields;
+  isLiveTailOnRef.current = isLiveTailOn;
+  liveTailTabIdRef.current = liveTailTabId;
+  liveTailNameRef.current = liveTailName;
 
   let minInterval = 'y';
   const findAutoInterval = (startTime: string, endTime: string) => {
     if (startTime?.length === 0 || endTime?.length === 0 || startTime === endTime) return 'd';
     const momentStart = dateMath.parse(startTime)!;
     const momentEnd = dateMath.parse(endTime)!;
     const diffSeconds = momentEnd.unix() - momentStart.unix();
+    // console.log("moment Start: ",momentStart);
```

Comment:
Let's remove these console log debug statements if there's no need for them anymore

Reply:
yeah okay, i'll remove them